language-server/vscoqtop/lspManager.ml

```diff
@@ -384,7 +390,7 @@ let coqtopInterpretToPoint params =
   match Hashtbl.find_opt states (DocumentUri.to_path uri) with
   | None -> log @@ "[interpretToPoint] ignoring event on non existant document"; []
   | Some { st; visible } ->
-    let (st, events, error_range) = Dm.DocumentManager.interpret_to_position st position ~should_block_on_error:!block_on_first_error in
+    let (st, events, error_range) = Dm.DocumentManager.interpret_to_position st position ~should_block_on_error:!block_on_first_error ~next_pos:!interp_to_next_position in
```

Comment:
From the previous comment, then instead of having this:
- do not modify ```Dm.DocumentManager.interpret_to_position```
- instead create a new function ```coqtopInterpretToNextSentence``` which does something among the lines of:
```ocaml
let coqtopInterpretToNextPoint params =
  let Notification.Client.InterpretToPointParams.{ textDocument; position } = params in
  let position = Dm.DocumentManager.get_position_of_next_sentence position in
  let uri = textDocument.uri in
  (*do all the same stuff as the other function*)
```
- implement the function ```get_position_of_next_sentence``` in the document manager